### PR TITLE
Fix flake8/mypy on CommonServerPython.py

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25125

## Screenshots
image

### CODE REVIEW

**Comment by Itay4** [comment]
@jochman looks ok
lets set to your sdk branch and verify lint passes as expected

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -367,8 +376,10 @@ def handle_proxy(proxy_param_name='proxy', checkbox_default_value=False, handle_
     if handle_insecure:
         if insecure_param_name is None:
             param_names = ('insecure', 'unsecure')
+        elif isinstance(insecure_param_name, str):
+            param_names = (insecure_param_name,)  # type: ignore
```
Comment (glicht) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:380]: Add the mypy exact error being ignored. See: https://xsoar.pan.dev/docs/integrations/linting#mypy

**Comment by jochman** [comment]
> The formatting is too much. Please revert formatting so it will be possible to review this with confidence. Use autopep8 for formatting.

For the protocol, it runs `autopep8` from the hooks.

File: Packs/Base/ReleaseNotes/1_0_5.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+- __CommonServerPython__
+-
```
Comment (glicht) [inline Packs/Base/ReleaseNotes/1_0_5.md:4]:
```suggestion
- Formatting and lint improvements.
```

File: Packs/Base/ReleaseNotes/1_0_6.md [hunk 3]
```diff
@@ -2,3 +2,4 @@
 #### Scripts
 - __CommonServerPython__
   - Added support for the CVE indicator class.
+  - Formatting and lint improvements.
```
Comment (glicht) [inline Packs/Base/ReleaseNotes/1_0_6.md:5]: This should be in a new release notes file 1_0_7.md. 1.0.6 is already released.

Comment (jochman) [reply Packs/Base/ReleaseNotes/1_0_6.md:5]: it's the result of multiple people working on the same pack. Maybe we should extract the CommonServer files into a separate pack.

**Comment by kirbles19** [comment]
@jochman Good, no changes.

**Comment by jochman** [comment]
The build isn't failing on CommonServerPython lints but on lints on other files, It'll be fixed [here](https://github.com/demisto/content/pull/7454).

Also, the error `NameError: name 'INTEGRATION_REGEX' is not defined` presented in the `Infrastructure Testing`, and `Collect Test List and Content Packs` is a problem with our Demisto-SDK master and not the pr itself.
An issue was opened [here](https://github.com/demisto/etc/issues/25431)
 
https://app.circleci.com/pipelines/github/demisto/content/14986/workflows/31aa6a93-61b0-45de-b522-7406e78a6fb2/jobs/56786/steps

File: Packs/Base/ReleaseNotes/1_0_7.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+### Scripts
+- __CommonServerPython__
```
Comment (glicht) [inline Packs/Base/ReleaseNotes/1_0_7.md:3]: use new format
```suggestion
#### CommonServerPython
```

**Comment by jochman** [comment]
Passed with custom branch -> https://app.circleci.com/pipelines/github/demisto/content/17839/workflows/8cf2e75e-7288-42b7-9346-a38a3041614d/jobs/60428/
Reverting the demisto-sdk branch back to the release. We can merge afterward.

**Pull request merged**